tbtaf/publisher/TBTAFPublisher.py

```diff
@@ -91,7 +99,12 @@ def PublishTestPlan(self, tBTestSuiteInstance, filePath, formatFlag):
         htmlString = htmlString.replace('r_report_time',s_reportTime)
         htmlString = htmlString.replace('r_tests',s_tests)
         
-        htmlFile.write(htmlString)
+        if formatFlag.lower() == "html":
+            htmlFile.write(htmlString)
+        else:
```

Comment:
not asumme that if !html then its gonna be PDF

Reply:
Fixed in commit c96db212103859741c85616c38c01e8ca5527fbd